cfme/tests/configure/test_workers.py

```diff
@@ -45,7 +47,10 @@ def test_restart_workers(appliance):
              message="Wait for all original workers are back online")
 
 
+@test_requirements.settings
 @pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1658373])
+@pytest.mark.meta(blocker=[1787350])
```

Comment:
These can be merged into a single meta line. The blocker meta list also requires `BZ` and since this is an automated test we can use `automates` rather than `coverage`. 
```suggestion
@pytest.mark.meta(blockers=[BZ(1787350)], automates=[1658373])
```

Reply:
cool, will do that :) but I found a bug/regression by automating a similar test, which was this test supposed to find, before my change. I will merge these 2 and this one will be fixed.